app/src/instant/java/com/omelan/cofi/utils/InsantUtils.kt

```diff
@@ -0,0 +1,18 @@
+package com.omelan.cofi.utils
+
+import android.app.Activity
+import android.content.Intent
+import com.google.android.gms.instantapps.InstantApps
+import com.omelan.cofi.BuildConfig
+
+fun showInstallPrompt(activity: Activity) {
+    val postInstall = Intent(Intent.ACTION_MAIN)
+        .addCategory(Intent.CATEGORY_DEFAULT)
+        .setPackage(BuildConfig.APPLICATION_ID)
+
+    InstantApps.showInstallPrompt(activity, postInstall, 1000, null)
+}
+
+fun isInstantApp() : Boolean {
+    return true
+}
```

Comment:
ktLint shows an error here: 

```
/home/runner/work/Cofi/Cofi/app/src/instant/java/com/omelan/cofi/utils/InsantUtils.kt:16:20: Unexpected spacing before ":" (colon-spacing)
```
so it should be:
```
fun isInstantApp(): Boolean {
    return true
}
```

but I would change it to: 

```suggestion
fun isInstantApp(context: Context) = context.packageManager.isInstantApp
```

as `instant` build variant doesn't guarantee that it's an instant app 😅